tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -108,3 +108,22 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search files inside the current folder
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world inside root" to "file1.txt"
+    And user "Alice" has created folder "/Folder"
+    And user "Alice" has uploaded file with content "hello world inside Folder" to "/Folder/file1.txt"
+    And user "Alice" has created folder "/Folder/SubFolder"
+    And user "Alice" has uploaded file with content "hello world inside SubFolder" to "/Folder/SubFolder/file1.txt"
+    When user "Alice" searches for "file1.txt" inside "/Folder" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these entries:
+      | /Folder/file1.txt           |
+      | /Folder/SubFolder/file1.txt |
```

Comment:
```suggestion
      | /Folder/file1.txt           |
      | /Folder/SubFolder/file1.txt |
```
 | file1.txt |
 |file1.txt  |
This will be also pass the test